system/lib/libc/musl/src/aio/aio_readwrite.c

```diff
@@ -20,7 +20,9 @@ static void notify_signal(struct sigevent *sev)
 		.si_pid = __pthread_self()->pid,
 		.si_uid = getuid()
 	};
+#ifndef __EMSCRIPTEN__
```

Comment:
Here also, we don't need to `#ifndef` this out, but let's have a no-op stub syscall for this.